siliconcompiler/toolscripts/rhel9/install-surelog.sh

```
@@ -5,12 +5,22 @@ set -ex
 # These dependencies are up-to-date with instructions from the INSTALL.md from the commit we are pinned to below
-sudo yum install -y gcc-toolset-12
-sudo dnf config-manager --set-enabled devel || true
-sudo yum install -y libuuid-devel java-11-openjdk-devel python3 zlib-static openssl-devel
-sudo dnf config-manager --set-disabled devel || true
-sudo yum install -y git
+install_prereqs gcc-toolset-12
+# The 'devel' repository is disabled by default. Enabling and disabling it is
+# root-only work whose only purpose is the install between them, so the whole
+# block is skipped when those packages are already present.
+devel_pkgs="libuuid-devel java-11-openjdk-devel python3 zlib-static openssl-devel"
+if prereqs_missing $devel_pkgs; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)